OpenDreamClient/EntryPoint.cs

```diff
@@ -18,6 +20,18 @@ public sealed class EntryPoint : GameClient {
         [Dependency]
         private readonly IDreamResourceManager _dreamResource = default!;
 
+        private const string UserAgent =
+            "Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 6.2; WOW64; Trident/7.0; .NET4.0C; .NET4.0E; .NET CLR 2.0.50727; .NET CLR 3.0.30729; .NET CLR 3.5.30729)";
+
+        public override void PreInit()
+        {
+            var config = IoCManager.Resolve<IConfigurationManager>();
+            if (config.GetCVar(OpenDreamCVars.SpoofIEUserAgent))
+            {
```

Comment:
```suggestion
            if (config.GetCVar(OpenDreamCVars.SpoofIEUserAgent)) {
```

Reply:
Fixed in [`7778d3c` (#1129)](https://github.com/OpenDreamProject/OpenDream/pull/1129/commits/7778d3cf9899af6aaf830a7de0b33ab247be76ee)